hbase-client/src/main/java/org/apache/hadoop/hbase/client/HBaseAdmin.java

```diff
@@ -1935,6 +1936,22 @@ protected Void postOperationResult(final Void result, final long deadlineTs)
     }
   }
 
+  private boolean isHexChar(char c) {
+    return (c >= 'A' && c <= 'F') || (c >= 'a' && c <= 'f') || (c >= '0' && c <= '9');
+
+  }
+  private boolean isMd5Hash(String encodedRegionName) {
```

Comment:
Yeah, this should be elsewhere too (RegionInfo as a static since that is where you get the MD5 length from?)... Capitalize the 'd'.... so MD5, not md5.

Reply:
Done. I put the new isMD5Hash() in RegionInfo and embed isHexDigit inside this method.